ocaml/xapi/vgpuops.ml

```diff
@@ -128,6 +127,7 @@ let reserve_free_virtual_function ~__context vm pf =
 
 let add_vgpus_to_vm ~__context host vm vgpus vgpu_manual_setup =
   (* Only support a maximum of one virtual GPU per VM for now. *)
+  try Db.VM.remove_from_other_config ~__context ~self:vm ~key:Xapi_globs.vgpu_pci with _ -> ();
```

Comment:
Oh oh... The () are needed here... Now everything below became part of the exception handler. @thomassa is going to fix it.